.ado/pipelines/templates/steps-parse-loadtest-output.yaml

```
@@ -23,7 +22,7 @@ steps:
           echo "*** Loading load test stats from pipeline artifacts $statsPath"
 
           $result = Compare-LocustStats -baselineFile $baselinePath -statsFile $statsPath -Verbose
```

```suggestion
          $result = Compare-LocustStats -baselineFile $baselinePath -statsFile $statsPath
```

@sebader now that you've switched from `Write-Verbose` to `Write-Host` we can drop `-Verbose` here.